tom_desc/fastdb_dev/DataTools.py

```diff
@@ -48,7 +48,9 @@
 from rest_framework.settings import api_settings
 
 _logger = logging.getLogger("fastdb_queries")
-_logout = logging.FileHandler( pathlib.Path( os.getenv('LOGDIR'), "/logs" ) / "fastdb_queries.log" )
+_log_path = pathlib.Path( os.environ.get('LOGDIR',"."), "/logs" )
```

Comment:
This change will mess up (at least) Liz's dev environment, where she bind-mounts a directory at `/logs` to hold the logs.

The docker-compose for the tests does create and mount something at /logs for (I think) all the necessary running services.

Reply:
Pulling out the "." default like we were talking about.